demisto_sdk/commands/unify/unifier.py

```diff
@@ -23,7 +23,7 @@
 
 PACK_METADATA_PATH = 'pack_metadata.json'
 CONTRIBUTOR_DISPLAY_NAME = ' ({} contribution)'
-CONTRIBUTOR_DETAILED_DESC = '### This is a {} contributed integration' \
+CONTRIBUTOR_DETAILED_DESC = '### {} contributed integration' \
```

Comment:
Let's capitalize this
```suggestion
CONTRIBUTOR_DETAILED_DESC = '### {} Contributed Integration' \
```